### PR TITLE
Check dependabot version update to vrms-data.yml 4829

### LINKED ISSUE #4829: Verify dependabot version updates to GHAs for vrms-data.yml
### Prerequisites
  1. You must be a member of Hack for LA to work on an issue. If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page.
  2. Please make sure you have read our Hack for LA [Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md) before you claim/start working on an issue.
### Overview
Our repo's [dependabot](https://github.com/hackforla/website/blob/gh-pages/.github/dependabot.yml) has been identifying the packages/dependencies for which a more recent version is available, and for each it has created a PR detailing which files in our codebase use that package/dependency. We need to check each of these files individually to determine whether we should or should not switch to the latest version. 
### Action Items
Refer to the [vrms-data.yml](https://github.com/hackforla/website/blob/gh-pages/.github/workflows/vrms-data.yml) file and review with the version updates recommended by the dependabot.
Select:
  - [x] PASSES TEST #4735 \`uses: actions/checkout@v2\` with \`uses: actions/checkout@v3\`
Or:
  - [ ] FAILS: #4735. Do not update.

Merge team:
- [x] Transfer results to #4815
### Resources/Instructions
- Never done GitHub actions? [Start here!](https://docs.github.com/en/actions)
- [Events that trigger workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows)
- [Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions)
- [actions/github-script](https://github.com/actions/github-script)

### PR DESCRIPTION
**Pull request by t-will-gillis**
Fixes #4829 

### What changes did you make and why did you make them ?

  - Tested updating the package version in the file `vrms-data.yml` per the dependabot suggestions:
    - Tested changing `uses: actions/checkout@v2` with `uses: actions/checkout@v3` per 4735
    - The test changing the `actions/checkout@v2` to `@v3` was successful- the vrms data was created. The previous 'node deprecation' warning is also removed. 
    
 ### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visual of current HfLA VRMS action</summary>

4829_vrms-data-hfla

</details>

<details>
<summary>Visual of successful test after version update</summary>
  
4829_vrms-data

</details>

### CODE REVIEW

**Comment by roslynwythe** [comment]
Availability: 6/20 2 - 4 pm
ETA 6/21 EOD

**Comment by LRenDO** [comment]
Availability: 6/20/23 11am-3pm
ETA: 6/21/23

File: .github/workflows/vrms-data.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ jobs:
     if: github.repository == 'hackforla/website'
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
-        token: ${{ secrets.HACKFORLA_BOT_PA_TOKEN }}
+        token: ${{ secrets.HACKFORLA_BOT_PA_TOKEN  }}
```
Comment (LRenDO) [inline .github/workflows/vrms-data.yml]: Replace `${{ secrets.HACKFORLA_BOT_PA_TOKEN  }}` with `token: ${{ secrets.HACKFORLA_BOT_PA_TOKEN }}`

**Review by LRenDO — DISMISSED** [review]
Hi @t-will-gillis! 

Your pull request looks great! It took me a bit to sort out, but after figuring out how to set up a PAT secret on my fork, I was able to verify that Update VRMS data GHA workflow ran successfully.

Some of the good work highlights:
- the issue was linked
- the to and from branches are correct
- the what is described
- workflow runs correctly

Minor, non-functional suggestions
- remove extra space before the last two curly braces in the line with the comment replace
```token: ${{ secrets.HACKFORLA_BOT_PA_TOKEN  }}``` with ```token: ${{ secrets.HACKFORLA_BOT_PA_TOKEN }}```
- add a  # in front of 4735 so that it reads `#4735` in the pull request so to make it easier to navigate to the page

Great work and thank you for your contribution!

**Review by roslynwythe — DISMISSED** [review]
The branches are setup correctly and the PR is descriptive.  I pushed the branch to my repo, made it default, edited the repo and and secret name in vrms-data.yml, then triggered the workflow manually and observed the updated time on [vrms_data.json](https://github.com/roslynwythe/website/blob/t-will-gillis-check-version-updates-4829/_data/external/vrms_data.json).  Great job!

File: .github/workflows/vrms-data.yml (hunk 1 above)
Comment (t-will-gillis) [reply .github/workflows/vrms-data.yml]: Hi @LRenDO - thanks for noticing the excess whitespace at the token, it is fixed now.

**Review by LRenDO — APPROVED** [review]
I see the minor edit fixing the extra space here. Looks great. Thanks @t-will-gillis!

**Review by roslynwythe — APPROVED** [review]
Approved.

**Pull request merged**